Running in wrapper mode for '66'

Running python job.

[name: "CPU-QUOTA"
int_val: 65
int_val: 41
int_val: 27
int_val: 31
int_val: 78
int_val: 83
int_val: 37
, name: "DISK"
int_val: 24
int_val: 40
int_val: 24
int_val: 33
int_val: 9
int_val: 32
int_val: 13
, name: "MEMORY"
int_val: 29
int_val: 28
int_val: 5
int_val: 23
int_val: 12
int_val: 25
int_val: 23
, name: "NET"
int_val: 33
int_val: 29
int_val: 13
int_val: 24
int_val: 11
int_val: 23
int_val: 8
]
The paramters are {u'CPU-QUOTA': array([65, 41, 27, 31, 78, 83, 37]), u'NET': array([33, 29, 13, 24, 11, 23,  8]), u'DISK': array([24, 40, 24, 33,  9, 32, 13]), u'MEMORY': array([29, 28,  5, 23, 12, 25, 23])}
{'13.56.14.158': ['haproxy', 'node-app', 'mongo'], '18.144.43.31': ['node-app', 'mongo']}
----------------------------------------
Problem running the job:
(<class 'paramiko.ssh_exception.BadAuthenticationType'>, BadAuthenticationType('Bad authentication type', [u'publickey']), <traceback object at 0x7fd41b75b248>)
Traceback (most recent call last):
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/home/ubuntu/throttlebot/src/spearmint/spearmint/runner.py", line 142, in run_python_job
    result = module.main(job.id, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 104, in main
    return explore_spearmint(workload_config, params)
  File "/home/ubuntu/throttlebot/src/spearmint/bayOptSearch/bayOpt.py", line 147, in explore_spearmint
    client = re.get_client(machine)
  File "/home/ubuntu/throttlebot/src/remote_execution.py", line 6, in get_client
    client.connect(ip, username="quilt", password="")
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 424, in connect
    passphrase,
  File "/home/ubuntu/.local/lib/python2.7/site-packages/paramiko/client.py", line 714, in _auth
    raise saved_exception
BadAuthenticationType: ('Bad authentication type', [u'publickey']) (allowed_types=[u'publickey'])
None
----------------------------------------
Job file reloaded.
Could not find value in output file.
Job failed in 0.45 seconds.
